docs/building/building-archlinux-template.md

```diff
@@ -1,152 +1,243 @@
+# These instructions are for Qubes 4.0.? and 4.1.?
 
-Archlinux template building instructions
-===========================================
```

Comment:
Could you please add the title back at the top, and move the note about Qubes versions below it?